hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -797,7 +797,8 @@ def _get_microscope_name(self, ImageTags):
             if ImageTags.Session_Info.Microscope != "[]":
                 return ImageTags.Session_Info.Microscope
         except AttributeError:
-            return ImageTags.Microscope_Info.Name
+            if 'Name' in ImageTags['Microscope_Info'].keys():
+                return ImageTags.Microscope_Info.Name
```

Comment:
Will the code calling `_get_microscope_name` work even if `_get_microscope_name` doesn't return anything?

Should there be:
```Python
else:
     return ''
```
?